packages/documentation/src/stories/icons/components/icon.docs.mdx

```diff
@@ -1,15 +1,17 @@
 import LinkTo from '@storybook/addon-links/react';
 import { Meta, Canvas, Story, Source, ArgsTable } from '@storybook/addon-docs';
 
-<Meta title="Components/Icon" />
+<Meta title="Icons/Icon" />
```

Comment:
For clarity.

```suggestion
<Meta title="Icons/Icon Component" />
```

Reply:
Should we rename the internet-header components (in the storybook navigation) accordingly?